### PR TITLE
Next frontend

### LINKED ISSUE #2: Selecting a transcript from the queue
As a GitHub-authenticated reviewer, I can select a transcript from the queue, and it will be automatically loaded into a transcript markdown editor.

### PR DESCRIPTION
**Pull request by Emmanuel-Develops**
Closes #2 

The original frontend still exists on the repo. This frontend has to be configured for docker and where it is hosted.

@nassersaazi I noticed the original frontend had netlify redirects config, kindly host this frontend with vercel, nextJs integrates better with vercel.

Also, the authentication is a test auth by creating a Github OAuth app with my account. When the project's OAuth account is ready we can replace it.

### CODE REVIEW

**Comment by nassersaazi** [comment]
i suggest you get rid of the original frontend directory altogether

**Review by nassersaazi — CHANGES_REQUESTED** [review]

File: frontend-next/yarn.lock [hunk 1]
```diff
@@ -0,0 +1,3671 @@
...
+
+yaml@^1.10.0:
+  version "1.10.2"
+  resolved "https://registry.yarnpkg.com/yaml/-/yaml-1.10.2.tgz#2301c5ffbf12b467de8da2333a459e29e7920e4b"
+  integrity sha512-r3vXyErRCYJ7wg28yvBY5VSoAF8ZvlcW9/BwUzEtUsjvX/DKs24dIkuwjtuprwJJHsbyUbLApepYTR1BN4uHrg==
+
+yocto-queue@^0.1.0:
+  version "0.1.0"
+  resolved "https://registry.yarnpkg.com/yocto-queue/-/yocto-queue-0.1.0.tgz#0294eb3dee05028d31ee1a5fa2c556a6aaf10a1b"
+  integrity sha512-rVksvsnNCdJ/ohGc6xgPwyN8eheCxsiLM8mxuE/t/mOVqJewPuO1miLpTHQiRgTKCLexL4MeAFVagts7HmNZ2Q==
```
[the remark this answers is not in the document]
Comment (Emmanuel-Develops) [reply frontend-next/yarn.lock:3671]: It helps maintain dependencies with yarn, we shouldn't ignore it

File: frontend-next/src/hooks/useTranscripts.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  //       toast({
+  //         title: "Unable to send invite",
+  //         description: errMessage,
+  //         status: "error",
+  //         duration: 6000,
+  //         isClosable: true,
+  //       });
+  //       return new Error(err)
+  //     });
+  // };
```
[the remark this answers is not in the document]
Comment (Emmanuel-Develops) [reply frontend-next/src/hooks/useTranscripts.ts:38]: I'll remove that

**Comment by adamjonas** [comment]
I'm on HEAD but getting an error

Screen Shot 2023-03-10 at 6 48 05 PM


Screen Shot 2023-03-10 at 6 47 12 PM

When I run on the daemon I get a CORS issue

Screen Shot 2023-03-10 at 6 50 50 PM

**Comment by Emmanuel-Develops** [comment]
I've fixed the error on HEAD

I can't recreate the CORS issue, ran it on `localhost:3001` and there was no issue

**Pull request merged**